pkg/kubeclient/kubelet_client.go

```diff
@@ -215,14 +215,14 @@ func (client *KubeletClient) HasCacheBeenUsed(host string) bool {
 	return false
 }
 
-//----------------- kubeletConfig -----------------------------------
+// ----------------- kubeletConfig -----------------------------------
 type KubeletConfig struct {
-	kubeConfig       *rest.Config
-	enableHttps      bool
-	allowTLSInsecure bool
-	port             int
-	timeout          time.Duration // timeout when fetching information from kubelet;
-	tlsTimeOut       time.Duration
+	kubeConfig           *rest.Config
+	enableHttps          bool
+	forceSelfSignedCerts bool
```

Comment:
I agree the original name `allowTLSInsecure` isn't easy to understand; I think it came from the `Insecure` flag in `TLSConfig` in `k8s.io/client-go`.  This new name `forceSelfSignedCerts` is far better, but still seems a little odd with the word "force".  Would `allowSelfSignedCerts` better?

Reply:
We don't "allow" anything. We force the self-signed certs usage, since we remove all the references to the certs we do have.